README.md

```diff
@@ -42,6 +42,7 @@ var browzine = {
   articleWording: "View Complete Issue",
   journalBrowZineWebLinkText: "Browse Now",
```

Comment:
> but I have a feeling this might have to do with testability of the code
They are different phrases. Summon "Browse Now", 360 Core "View Journal in BrowZine".

`summonJournalBrowZineWebLinkText` kinda want to do this, just concerned if any customers are using the old property, I might be able to layer the use like: `var text = journalBrowZineWebLinkText || summonJournalBrowZineWebLinkText || "default text"` or something to be backwards compatible. Though it's not like a breaking change because we use default text anyway...

Let me think about that.

Reply:
Yeah, I like the `||` of those strings 👍